lib/rubocop/cop/rspec/described_class_module_wrapping.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Avoid opening modules and defining specs within them.
+      #
+      # @example
+      #   # bad
+      #   module MyModule
+      #     RSpec.describe MyClass do
+      #       # ...
+      #     end
+      #   end
+      #
+      #   # good
+      #   RSpec.describe MyModule::MyClass do
+      #     # ...
+      #   end
+      #
+      # @see https://github.com/rubocop-hq/rubocop-rspec/issues/735
+      class DescribedClassModuleWrapping < Cop
+        MSG = 'Avoid opening modules and defining specs within them.'
+
+        def_node_matcher :rspec_block?,
+                         RuboCop::RSpec::Language::ExampleGroups::ALL.block_pattern # rubocop:disable Metrics/LineLength
```

Comment:
`RuboCop::Cop::RSpec::Cop` includes `RuboCop::RSpec::Language` so this might be just `ExampleGroups::ALL.block_pattern`.

Reply:
Done in 44baef49aba7c814d30aa1c3800da032e4b527dc. Thanks for the quick review!